packages/labs/scoped-registry-mixin/README.md

```diff
@@ -0,0 +1,43 @@
+# @lit-labs/scoped-registry-mixin
+
+`UseScopedRegistry` mixin for `LitElement` that integrates with the speculative [Scoped CustomElementRegistry polyfill](https://github.com/webcomponents/polyfills/tree/master/packages/scoped-custom-element-registry).
+
+🚨 **Warning: Scoped Custom Element Registries is a speculative browser API that has not shipped and may change.** This mixin is provided to evaluate the spec proposal and provide feedback. **Use this feature and the polyfill in production code at your own risk.**
```

Comment:
Rather than "speculative browser API" I'd say that it's a proposal.

```suggestion
🚨 **Warning: Scoped Custom Element Registries is a proposed browser API. It has not been finalized nor shipped in any browser.** This mixin is provided to evaluate the proposal and facilitate feedback. **Use this feature and the polyfill in production code at your own risk.**
```

Reply:
done